aksel.nav.no/website/components/website-modules/ConsentBanner.tsx

```diff
@@ -0,0 +1,137 @@
+"use client";
+
+import Script from "next/script";
+import { useEffect, useRef, useState } from "react";
+import { Cookies } from "typescript-cookie";
+import { FaceLaughIcon } from "@navikt/aksel-icons";
+import {
+  BodyLong,
+  Button,
+  Detail,
+  HStack,
+  Link,
+  Modal,
+} from "@navikt/ds-react";
+
+const CONSENT_TRACKER_ID = "acceptTracking";
+
+type CONSENT_TRACKER_STATE = "undecided" | "accepted" | "rejected";
+
+export const getStorageAcceptedTracking = () => {
+  const rawState = Cookies.get(CONSENT_TRACKER_ID);
+  if (!rawState) {
+    return "undecided";
+  }
+
+  return rawState as CONSENT_TRACKER_STATE;
+};
+
+export const setStorageAcceptedTracking = (state: CONSENT_TRACKER_STATE) => {
+  Cookies.set(CONSENT_TRACKER_ID, state, { expires: 365 });
+};
+
+export const ConsentBanner = () => {
+  const refUmamiTag = useRef<string>("");
+
+  useEffect(() => {
+    const consentAnswer = getStorageAcceptedTracking();
+    if (consentAnswer === "undecided") {
+      ref.current?.showModal();
+    }
+
+    const isProdUrl = () => window.location.host === "aksel.nav.no";
```

Comment:
Think it would make sense to extract this to a util function `getCurrentEnviroment`or something. Think we should account for if `example` or `template` is viewed to, so
```
const isProdUrl = () => window.location.host === "aksel.nav.no";
    const isExample = () => window.location.pathname.includes("eksempler/");
    const isTemplate = () => window.location.pathname.includes("templates/");
    const isPreview = () => !!document.getElementById("exit-preview-id");
```

Might also make sense to turn of tracking when on example/template 🤔 If you visit a component-page, we dont want to show that as extra view when the example loads (umami might account for this, so no idea if needed)

Reply:
et ish problem med Umami tags er at det ser ut som det bare kan være _en_ tag om gangen? 💭

Reply:
I guess it's another point in favor of tracking manually curated objects 
![image](https://github.com/user-attachments/assets/7a962652-a796-4238-b784-3f47340a5f54)

Reply:
Think I've got a good solution now for using the auto tracker + setting a useful tag.

Landing on two tags: `organic` (for natural organic traffic we care about), and `artificial` (which might still be useful, but require a bit more effort to filter out later, usually I think we can do a very simple filter on a path matcher `/admin/` for example, to see how much traffic there is in sanity studio or similar.)